toolchains/BUILD.bazel

```diff
@@ -10,3 +14,10 @@ toolchain(
     toolchain = ":protoc_java_toolchain",
     toolchain_type = "@rules_java//java/proto:toolchain_type",
 )
+
+# Sources needed for release tarball.
+filegroup(
```

Comment:
Instead of using Bazel to produce the tarball, you could use `git archive`.

Advantages:
- you never forget to include some files
- remove a bunch of BUILD file content related to creating the packaging
- drop the dependency on rules_pkg, it's dragging you down and has bugs like bringing in that broken rules_python
- it matches what GitHub serves when a user switches to a pre-release (e.g. fetches an archive from a git sha at HEAD)

https://github.com/bazel-contrib/rules-template documents the pattern.

Reply:
Interesting thought! It was not possible before Stardoc 0.7 because we did not want to force the dep on io_bazel onto all Stardoc users. But it could be viable now.

Reply:
Fixed: f35a12ecb346891b14dae2738e816921ef0437ea. Thanks for the suggestion!